nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-bulk-contrib.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.imaging" version="1.0.0">
```

Comment:
I would add `.bulk` to the component name to avoid having a too much generic component name.